tests/main/apparmor-batch-reload/task.yaml

```diff
@@ -10,24 +10,30 @@ details: |
     and then verifies apparmor cache to ensure expected profiles were created by fallback
     logic.
 
-systems: [ubuntu-1*]
```

Comment:
What is the reasoning behind this change?

Reply:
The test was running on 14/16/18 only, but IMO we do not need the test to run on those old releases, but also not on every possible release, so I picked 20 and 24 as the two extremes from the range of 'modern' releases.

Reply:
I see, a comment here will be nice for future reference